ext/File-Find/lib/File/Find.pm

```diff
@@ -1,21 +1,21 @@
 package File::Find;
+
 use 5.006;
 use strict;
 use warnings;
 use warnings::register;
-our $VERSION = '1.43';
+
 use Exporter 'import';
 require Cwd;
+require File::Basename;
```

Comment:
Not sure why it's using `require` instead of `use`, just to avoid imports (which could be achieved by passing empty list to `use` as well)?

Reply:
Let's use `git blame` to investigate.  For line 9 above, that points to this commit:
```
commit 6280b799bc1eabee6c39553ae157386070a217cd
Author:     unknown <perl5-porters.nicoh.com>
AuthorDate: Wed Jan 10 01:46:24 1996 +0000
Commit:     Andy Dougherty <doughera@lafayette.edu>
CommitDate: Wed Jan 10 01:46:24 1996 +0000

    Fixed exporting of symbols to work.

diff --git a/lib/File/Find.pm b/lib/File/Find.pm
index c151bcc891..780dfea469 100644
--- a/lib/File/Find.pm
+++ b/lib/File/Find.pm
@@ -1,9 +1,10 @@
 package File::Find;
 require 5.000;
 require Exporter;
-use Config;
-use Cwd;
-use File::Basename;
+require Config;
+require Cwd;
+require File::Basename;
...
```
Now, the commit message doesn't fully describe the context that led to this commit, but, IMO, if it was good for Andy Dougherty (@doughera88) in 1996, it's still good for me today!